installers/helm/templates/postgres-operator-install.yaml

```diff
@@ -0,0 +1,13 @@
+{{ $_ := set . "deployAction" "install" }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: pgo-deploy
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "postgres-operator.labels" . | indent 4 }}
+  annotations:
+    helm.sh/hook: post-install
+    helm.sh/hook-delete-policy: hook-succeeded,before-hook-creation
```

Comment:
I added `before-hook-creation` here to help if the job is run but fails. In that case the resources are not cleaned up and the install cannot be run again. By using `hook-succeeded` and  `before-hook-creation` the failed job will be kept around until a new command is run allowing for the user to look at the failed logs. Would this be good to document?

Reply:
Yes, users should know what happens in the success and failure case.